components/pkg-export-docker/defaults/Dockerfile.hbs

```diff
@@ -23,10 +23,6 @@ ENV HAB_NON_ROOT=1
 
 USER {{primary_user_id}}:{{primary_group_id}}
 
-{{#each volumes}}
```

Comment:
We may want to flag this in release notes as there is a bit of a behavior change, but for the better all-around, /cc @tashimi